README.md

```diff
@@ -39,11 +39,11 @@ The checks present in the package:
 
   Only one *"~V" *section can occur in an LAS 2.0 file.
 
-  ~V section must contain the following lines.
+  ~V section must contain the following lines: VERS, WRAP.
```

Comment:
~V section must contain the lines: VERS, WRAP.

Please remove the word following